tests/unittest/llmapi/apps/_test_disagg_serving_multi_nodes.py

```
@@ -0,0 +1,219 @@
+        with RemoteOpenAIServer(model_path,
+                                port=GEN_SERVER_PORT,
+                                cli_args=args,
+                                host=host,
+                                env=env(),
+                                rank=RANK % SLURM_NTASKS_PER_NODE,
+                                extra_config=extra_config) as server:
+            yield server
+
+
+@pytest.fixture(scope="module")
+def disagg_server(worker: RemoteOpenAIServer):
```

Ruff (0.12.2)

Unused function argument: `worker`

(ARG001)